great_expectations_cloud/agent/agent.py

```diff
@@ -102,6 +111,82 @@ class Config:
         json_loads = orjson_loads
 
 
+# Use Retries = 10, --workers 1
+# Data Context as a Dependency
+async def handler(msg: dict, gx_agent: GXAgent, correlation_id: str) -> None:
+    print(f"Received: {msg}")
+    print(f"GX Agent: {gx_agent}")
+
+    event = EventHandler.parse_event_from_dict(msg)
+    event_context = EventContext(
+        event=event,
+        correlation_id=correlation_id,  # msg.correlation_id,
+        processed_successfully=mock.Mock(),
+        processed_with_failures=mock.Mock(),
+        redeliver_message=mock.Mock(),
```

Comment:
Why are the values here set to `mock.Mock()`?

Reply:
+1